source/documentation/monitoring-alerts/bind-exporter-to-prometheus.md

```diff
@@ -39,7 +39,7 @@ Within 10 minutes Prometheus will start scraping your application for metrics, y
 
 #### IP Whitelist your applications metrics endpoint
 
-If you're using a [blue-green deployment process][] with a zero downtime plugin such as [autopilot][] you should disable the basic auth on the metrics endpoint when using the [Ruby gem][] or [Python library][] and instead protect the metrics endpoint using IP whitelisting in order to minimise gaps in metrics between deployments.
+If you're using a [blue-green deployment process][] with a zero downtime plugin such as [autopilot][] and you don't wish to publish metrics publicly you should use an IP whitelist instead of basic auth on the metrics endpoint in order to minimise gaps in metrics between deployments.
```

Comment:
```suggestion
If you're using a [blue-green deployment process][] with a zero downtime plugin such as [autopilot][] and you don't wish to publish metrics publicly, you should use an IP whitelist instead of basic auth on the metrics endpoint in order to minimise gaps in metrics between deployments.
```